src/styles/layouts/_login.scss

```diff
@@ -74,6 +80,12 @@
     }
   }
 
+  @media screen and (max-width: 270px) {
```

Comment:
Bez przesady ;p 320px to jest minimalna wartość, przy której aplikacja powinna wyglądać estetycznie. Tutaj natomiast osobiście zastosowałbym `340px` ;)

Reply:
Ok 330px niech bedzie.